data/sql/updates/pending_db_world/rev_1601315140580867400.sql

```diff
@@ -0,0 +1,125 @@
+(11915,0,2,'No kill me! No kill me!',12,0,100,0,0,0,1863,0,'Gogger Rock Keeper');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=4001;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=4001 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4001,0,0,0,0,0,100,0,4000,11000,7000,15000,11,8139,32,0,0,0,0,2,0,0,0,0,0,0,0,'Windshear Tunnel Rat - In Combat - Cast Fevered Fatigue');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=11917;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(11917,0,0,'You NO take candle!',12,0,100,0,0,0,16658,0,'Gogger Geomancer'),
+(11917,0,1,'Yiieeeee! Me run!',12,0,100,0,0,0,1864,0,'Gogger Geomancer'),
+(11917,0,2,'No kill me! No kill me!',12,0,100,0,0,0,1863,0,'Gogger Geomancer');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=4002;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=4002 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4002,0,0,0,2,0,100,1,0,15,0,0,25,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Windshear Stonecutter - Between 0-15% Health - Flee For Assist (No Repeat)');
+
+UPDATE `creature_template` SET `AIname`='SmartAI' WHERE `entry`=11912;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=11912 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(11912,0,0,0,4,0,100,0,0,0,0,0,11,6268,0,0,0,0,0,2,0,0,0,0,0,0,0,'Grimtotem Brute - On Aggro - Cast RushIng Charge'),
+(11912,0,1,0,2,0,100,1,0,15,0,0,25,1,0,0,0,0,0,1,0,0,0,0,0,0,0,'Grimtotem Brute - Between 0-15% Health - Flee For Assist (No Repeat)');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=17236; CHECk
```

Comment:
mistake

Reply:
Its was about last part to add despawn on evade but forgot to remove check xP